js-src/packages/scripting-editor/app/static-completion-provider.ts

```diff
@@ -0,0 +1,111 @@
+import * as monaco from "monaco-editor";
+
+export type StaticCompletionItem = {
+  functionName: string;
+  arguments: string;
```

Comment:
Could there be constants in the completion items, that have no arguments?
Maybe:
- Make `arguments` optional (looking at the code this seems to be handled in most (if not all) cases already)
- Do not add parentheses if `arguments` is undefined

Reply:
Good point. I had the same thought this night: how about adding the constants, as well as available columns and flow variables? And for that we need to be able to drop the arguments